test/utils/devices/devices.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2022 The Kubernetes Authors.
```

Comment:
this copyright boilerplate is not really correct. Not wrong either, but we should use the Red Hat copyright for this code. For example, see: https://github.com/openshift-kni/numaresources-operator/blob/v0.4.12-0.2022092201/test/e2e/serial/tests/workload_placement_taint.go#L2

And yes, this means we should fix quite some copyrights boilerplates around the codebase. So let's make sure the new code is correct

Reply:
Thanks for raising this, do we care about the year?

Reply:
you're right, I picked a random examples. In this case 2021 seems the correct choice if the code was written in 2021

Reply:
in general: when we do code movement, we keep the older copyright year (obviously); when we write brand new code, we use the current year.